### PR TITLE
[LLT-5479] Redirect go logs to Telio logger

### PR DESCRIPTION
**Pull request by stalowyjez**
### Problem
There are some logs logged by wireguard-go (especially wintun) which are not forwarded to libtelio logger

### Solution
Log the golang default logger output with libtelio logger, so these logs are not lost

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: wireguard-go-rust-wrapper/wireguard-go/main.go [hunk 1]
```diff
@@ -97,6 +98,13 @@ func tracef(format string, args ...interface{}) {
 	trace_logger.Printf(format, args...)
 }
 
+type DevLoggerWriter struct {}
+
+func (t DevLoggerWriter) Write(p []byte) (n int, err error) {
+	debugf(string(p[:]))
```
Comment (matislovas) [inline wireguard-go-rust-wrapper/wireguard-go/main.go:104]: Does this only have a debug logs? Or can we differentiate?

Comment (stalowyjez) [reply wireguard-go-rust-wrapper/wireguard-go/main.go:104]: I'm afraid that it provides only redirection.  It seems that wintun has only INFO and ERROR logs and I felt like making them DEBUG makes some sense, replacing both of them with DEBUG sounds better to me than making INFO -> ERROR or ERROR -> INFO (they are not our logs and we add them now mostly for debugging purposes anyway).

Comment (matislovas) [reply wireguard-go-rust-wrapper/wireguard-go/main.go:104]: Yeah, maybe we can leave the `debug` only, but maybe than mark them as `wintun-log` or sth similar? Do distinguish them from the rest of the logs.

Comment (stalowyjez) [reply wireguard-go-rust-wrapper/wireguard-go/main.go]: @matislovas added some tag

**Review by mathiaspeters — APPROVED** [review]
+1

**Pull request merged**